crates/ruff_python_formatter/src/string/docstring.rs

```diff
@@ -1538,7 +1541,7 @@ fn needs_chaperone_space(normalized: &NormalizedString, trim_end: &str) -> bool
 /// to the next multiple of 8. This is effectively a port of
 /// [`str.expandtabs`](https://docs.python.org/3/library/stdtypes.html#str.expandtabs),
 /// which black [calls with the default tab width of 8](https://github.com/psf/black/blob/c36e468794f9256d5e922c399240d49782ba04f1/src/black/strings.py#L61).
-fn indentation_length(line: &str) -> TextSize {
+fn indentation_length(line: &str) -> u32 {
```

Comment:
If this is meant to be a byte offset, then I think this should be a `usize`. And then you can get rid of the `as usize` conversions above.

But if it's a byte offset, then what about the tab business going on in here? I think that makes this not a byte offset but maybe something else?

Clearer docs on this function might help.

Reply:
Yes, I updated it to use `usize`. 

The point of this PR is that `indentation_length` **does not** return a byte offset. It's the length/width in columns (or spaces). That's why we shouldn't use `TextSize` or `TextRange`, which we exclusively use for byte offsets.